### PR TITLE
admin view and add user

### PR DESCRIPTION
**Pull request by hangxuka**

### CODE REVIEW

File: app/views/admin/users/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            <%= link_to image_tag(Settings.admin.users.url,
+              size: Settings.admin.users.size), admin_users_path %>
+          <% end %>
+        </td>
+        <td>
+          <%= link_to t("admin.users.table.edit"), "#",
+            class: "btn btn-primary" %>
+          <%= link_to t("admin.users.table.delete"), "#",
+            method: :delete, data: {confirm: t("admin.messages.confirm")},
+              class: "btn btn-danger" %>
```
Comment (TaoLuyen) [inline app/views/admin/users/index.html.erb]: lui vao 1 lan thoi

File: config/routes.rb [hunk 2]
```diff
@@ -13,6 +13,7 @@
     resources :categories
     resources :books
     resources :requests, only: [:index]
+    resources :users
```
Comment (TaoLuyen) [inline config/routes.rb]: dung cai nao thi only cai day.

File: app/controllers/admin/users_controller.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
+class Admin::UsersController < ApplicationController
+
```
Comment (TaoLuyen) [inline app/controllers/admin/users_controller.rb]: kiểm tra quyền

**Comment by VuDinhHao** [comment]
ok

**Comment by TaoLuyen** [comment]
ok

**Pull request merged**